runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -137,6 +138,18 @@
     // A future to track the last submitted read request
     volatile private Future lastReadRequest;
 
+    /**
+     * We can add streams to be ignored during the
+     * reconstruction of the state (e.g. raw streams)
+     */
+    @Getter
+    private Set<UUID> streamsToIgnore = new HashSet<>();
+
+    /**
+     * We can register streams with non-default type
+     */
+    private final Map<UUID, Builder> customTypeStreams = new HashMap<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651447)